core/src/main/java/com/linecorp/armeria/common/logging/RequestLog.java

```diff
@@ -621,22 +609,4 @@ String toStringResponseOnly(Function<? super HttpHeaders, ?> headersSanitizer,
     String toStringResponseOnly(Function<? super ResponseHeaders, ?> headersSanitizer,
                                 Function<Object, ?> contentSanitizer,
                                 Function<? super HttpHeaders, ?> trailersSanitizer);
-
-    /**
-     * {@inheritDoc}
-     *
-     * @deprecated Use {@code RequestLog.context().attr()}.
```

Comment:
Did we already release armeria with these deprecations? If not, then this will be breaking with deprecation time. Maybe it's such a minor API it's ok to do that though.

Reply:
We did not release yet. Perhaps we can release another version before merging this?